Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -8,22 +8,22 @@
 
 namespace Xamarin.Forms
 {
-	public abstract class Cell : Element, ICellController, IElementConfiguration<Cell>, IFlowDirectionController, IPropertyPropagationController, IVisualController
+	// Don't add IElementConfiguration<Cell> because it kills performance on UWP structures that use Cells
+	public abstract class Cell : Element, ICellController, IFlowDirectionController, IPropertyPropagationController, IVisualController
```

Comment:
so, this is technically a breaking change ? can you investigate if we should care about that or not ?

Reply:
@StephaneDelcroix what are the scenarios where you would see this cause breaking? I just want to make sure I've considered them all.

The main case I know of would be if in user code someone casted Cell as IElementConfiguration<Cell> and is using that instance to apply the platform specifics. 

I tested if it would cause breaking if someone inherited from Cell
```C#
   // this is compiled against a cell with the interface
    public class ShaneCell : Cell
    {
    }
```

I then used the compiled library in a project that has the interface removed. 
I also tested visa versa. 

This fixes performance on UWP specifically with listview and since it seems like a minor 4.0 breaking change then I think we're ok